vendors/sensecap/drivers/s2101/driver.yaml

```diff
@@ -0,0 +1,2 @@
+# A reference to the corresponding driver on our private drivers repository
+protocolId: sensecap:air-th:1
```

Comment:
you have added protocolId to link the device to a driver, However, there is no driver for sensecap s2101 on our private device catalog. Either you add the package under /drivers/s2101 and thus will be public, or you contact @norbertherbert to send the private driver package then we add it on our private catalog. 

By the way, I can see on our private driver catalog that a driver exist with a protocolId "os1-seap:air-th:1", and the model already exist here https://github.com/actility/device-catalog/tree/main/vendors/sensecap/models/air-th .

Is it the same device and you prefer modifying the name, id, photo, etc... ? Or it is a new one?

Reply:
Hello @mostafaibr ,

It seems the driver is optional, so could we upload without the driver files at the beginning? Cause we don't have one(decoder files) for Actility yet, but we'll put this on the list and upload it in future. So can you tell me what we could do now?
Besides, yes, it's a new one an upgrade version of the previous one.

Reply:
@Kway99 ,

In that case, you have to remove the directory /drivers/s2101. 
and then remove the protocolId property from /modes/s2101/model.yaml